lib/ansiblelint/errors.py

```diff
@@ -0,0 +1,32 @@
+class Match(object):
```

Comment:
Explicit inheritance from object is unnecessary in py3
```suggestion
class Match:
```

Reply:
And while you're here plz add a class docstring.